tcl/windows/browser.tcl

```diff
@@ -4,6 +4,11 @@
 
 namespace eval ::gbrowser {}
 
+proc ::gbrowser::resizeBoard { n s } {
+    set w .gb$n
```

Comment:
I understand that the other gbrowser::functions do this, but resetting the window name in every function is wrong.